python/pyspark/sql/column.py

```diff
@@ -54,6 +54,22 @@ def _to_java_column(col):
     return jcol
 
 
+def _to_sorted_java_columns(cols, ascending=True):
+    if len(cols) == 1 and isinstance(cols[0], list):
+        cols = cols[0]
+    jcols = [_to_java_column(c) for c in cols]
+    if isinstance(ascending, (bool, int)):
+        if not ascending:
+            jcols = [jc.desc() for jc in jcols]
+    elif isinstance(ascending, list):
+        jcols = [jc if asc else jc.desc()
+                 for asc, jc in zip(ascending, jcols)]
```

Comment:
Btw this can be easily worked around. I think we don't need to add this argument